pkg/api/types.go

```diff
@@ -3023,6 +3023,10 @@ type NodeList struct {
 type NamespaceSpec struct {
 	// Finalizers is an opaque list of values that must be empty to permanently remove object from storage
 	Finalizers []FinalizerName
+
+	// NetworkPolicy describes the use of NetworkPolicy objects in this namespace. If not specified, then default policies will be applied and NetworkPolicy objects in the Namespace will be ignored
+	// +optional
+	NetworkPolicy *NamespaceNetworkPolicy
```

Comment:
Namespace is in the innermost/lowermost layer, which I've been calling the Nucleus.

NetworkPolicy is part of the governance layer.

I don't see any policies being added to Namespace.

Layering breakdown (join kubernetes-dev@googlegroups.com to view):
https://docs.google.com/presentation/d/1GpELyzXOGEPY0Y1ft26yMNV19ROKt8eMN67vDSSHglk/edit#slide=id.g1dd3a89dc7_0_5